Makefile

```diff
@@ -112,6 +112,18 @@ pause-container-release: pause-container
 	mkdir -p "$(shell pwd)/out"
 	@docker save ${PAUSE_CONTAINER_IMAGE}:${PAUSE_CONTAINER_TAG} > "$(shell pwd)/out/${PAUSE_CONTAINER_TARBALL}"
 
+# Variable to determine branch/tag of amazon-ecs-cni-plugins
+revision=master
+cni-sources:
+	@git clone https://github.com/aws/amazon-ecs-cni-plugins.git --branch $(revision)
+
+cni-plugins: cni-sources
```

Comment:
Yeah, as explained in the previous comment, that's more favorable to having to do manual overrides during development process.